packages/metatransaction-broadcaster/MetatransactionBroadcaster.js

```diff
@@ -215,25 +176,50 @@ class MetatransactionBroadcaster {
         valid = await this.isAddressValid(tx.args[1]);
       } else if (tx.signature === "approve(address,uint256)") {
         valid = await this.isAddressValid(tx.args[0]);
+      } else if (tx.signature === "setAuthority(address)") {
+        // Get the most recent metatx this user sent on colonyNetwork
```

Comment:
This should be put into it's own function to be consistent with the style of the other branches